src/dm_ls_scf_methods.F

```diff
@@ -454,28 +456,33 @@ END SUBROUTINE density_matrix_sign
 !> \param matrix_s ...
 !> \param matrix_s_inv ...
 !> \param threshold ...
+!> \param sign_symmetric ...
+!> \param submatrix_sign_method ...
 !> \par History
 !>       2010.10 created [Joost VandeVondele]
 !> \author Joost VandeVondele
 ! **************************************************************************************************
    SUBROUTINE density_matrix_sign_fixed_mu(matrix_p, trace, mu, sign_method, sign_order, matrix_ks, &
-                                           matrix_s, matrix_s_inv, threshold)
+                                           matrix_s, matrix_s_inv, threshold, sign_symmetric, submatrix_sign_method)
 
       TYPE(dbcsr_type), INTENT(INOUT)                    :: matrix_p
       REAL(KIND=dp), INTENT(OUT)                         :: trace
       REAL(KIND=dp), INTENT(INOUT)                       :: mu
       INTEGER                                            :: sign_method, sign_order
       TYPE(dbcsr_type), INTENT(INOUT)                    :: matrix_ks, matrix_s, matrix_s_inv
       REAL(KIND=dp), INTENT(IN)                          :: threshold
+      LOGICAL                                            :: sign_symmetric
+      INTEGER                                            :: submatrix_sign_method
 
       CHARACTER(LEN=*), PARAMETER :: routineN = 'density_matrix_sign_fixed_mu', &
          routineP = moduleN//':'//routineN
 
       INTEGER                                            :: handle, unit_nr
+      LOGICAL                                            :: converged = .FALSE., symmetrize = .TRUE.
```

Comment:
Unfortunately, in Fortran inline initialization likes these will implicitly add the `SAVE` attribute. This makes the variable persistent across invocations and thread-shared. The solution is to simply put the initialization on a separate line.